src/Converter/LineItemsConverterInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Converter;
+
+use Doctrine\Common\Collections\Collection;
+use Sylius\RefundPlugin\Entity\LineItemInterface;
+use Sylius\RefundPlugin\Model\UnitRefundInterface;
+
+interface LineItemsConverterInterface
+{
+    /**
+     * @param array<UnitRefundInterface> $units
+     *
+     * @return Collection|LineItemInterface[]
+     */
+    public function convert(array $units): Collection;
```

Comment:
Is there any reason why we need to return a Doctrine's Collection? Could it be just a plain old PHP array?

Reply:
There is no specific reason, a line item is just an entity now, so using a collection here seems to be appropriate imo

Reply:
If it could be an array I would vote for this 👍 Collection seems a little bit coupled with the ORM and entities, this interface could be used anywhere else. We can still just use the result of this function as an `ArrayCollection` argument 💃